pkg/debian/postinst

```diff
@@ -1,23 +1,41 @@
 #!/bin/sh
-update-rc.d $PKG_NAME defaults 91 09
+
+if [ -f /etc/init.d/$PKG_NAME ] ; then
+  update-rc.d $PKG_NAME defaults 91 09
+fi
 
 mkdir -p /var/lib/$PKG_NAME
 mkdir -p /etc/$PKG_NAME.conf.d
 
-case "$1" in
-    configure)
-         /etc/init.d/$PKG_NAME stop || :
+legacy_start() {
+  /etc/init.d/$PKG_NAME stop || :
+
+  # Cleanup init's mess if it fails to stop
+  PIDFILE=/var/run/$PKG_NAME.pid
+  start-stop-daemon --stop --retry 5 --quiet --pidfile $PIDFILE --name rackspace-monit --signal KILL
 
-         # Cleanup init's mess if it fails to stop
-         PIDFILE=/var/run/$PKG_NAME.pid
-         start-stop-daemon --stop --retry 5 --quiet --pidfile $PIDFILE --name rackspace-monit --signal KILL
+  # Start only if we find a config file in place
+  if [ -f /etc/$PKG_NAME.cfg ]; then
+    /etc/init.d/$PKG_NAME start || :
```

Comment:
i have some reservations about this, if we don't start b/c there is no config file, how does the user know that?  At least if the agent starts w/o a config file the logs show that.